assets/scss/components/_forms.scss

```diff
@@ -149,3 +149,78 @@ Faux inputs visually mimic actual input elements.
 		}
 	}
 }
+
+$toggleSize: 20px;
+$C_toggleHighlight: $C_lightPurple; // replace $C_lightPurple with $C_blue once we start to use `swarm-constants`
+
+.toggleLabel {
```

Comment:
not really, although `toggleLabel-container` does not need to be and probably shouldn't be nested in `toggleLabel`.